### PR TITLE
PWA-712: Fix Chrome prev / next carousel button placement

### PR DESCRIPTION
**Pull request by davemacaulay**
## Description
They should be in the center of the image but they are instead at the top of the image:
https://i.gyazo.com/98b75a0fb607669e627f2d1a3091b415.gif

## Related Issue
https://jira.corp.magento.com/browse/PWA-712

### Verification Stakeholders
@jimbo 
@sirugh 

### Verification Steps
1. Check previous / next in Chrome mobile

## Screenshots / Screen Captures (if appropriate)
https://i.gyazo.com/98b75a0fb607669e627f2d1a3091b415.gif

## Checklist

- [x] I have added tests to cover my changes, if necessary.
- [x] I have updated the documentation accordingly, if necessary.

### CODE REVIEW

**Comment by devops-pwa-codebuild** [comment]
**Performance Test Results**

The following fails have been reported by WebpageTest. These numbers indicates a possible performance issue with the PR which requires further manual testing to validate.

https://pr-2512.pwa-venia.com : LH Performance Expected 0.85 Actual 0.57, LH Best Practices Expected 1 Actual 0.92
https://pr-2512.pwa-venia.com/venia-tops.html : LH Performance Expected 0.75 Actual 0.36, LH Best Practices Expected 1 Actual 0.92
https://pr-2512.pwa-venia.com/valeria-two-layer-tank.html : LH Performance Expected 0.8 Actual 0.49, LH Accessibility Expected 0.9 Actual 0.89, LH Best Practices Expected 1 Actual 0.92

File: packages/venia-ui/lib/components/ProductImageCarousel/carousel.css [hunk 1]
```diff
@@ -45,6 +45,18 @@
...
 
+.chevron {
+    composes: root from '../Icon/icon.css';
+    align-self: center;
+    border: 2px solid transparent;
+}
+
+.nextButton:focus > .chevron,
+.previousButton:focus > .chevron {
+    box-shadow: 0 0 4px 2px rgb(var(--venia-global-color-teal));
```
Comment (sirugh) [inline packages/venia-ui/lib/components/ProductImageCarousel/carousel.css:56]: CSS tokens don't exist on the release branch. I'm fine leaving this as is though, it'll just start working on `develop` and in the next branch.

Comment (jimbo) [reply packages/venia-ui/lib/components/ProductImageCarousel/carousel.css:56]: Probably worth changing now.

Comment (davemacaulay) [reply packages/venia-ui/lib/components/ProductImageCarousel/carousel.css]: Updated to use `--venia-teal` good catch @sirugh

**Comment by dpatil-magento** [comment]
QA Approved.

**Pull request merged**